### PR TITLE
Makefile, README, Clean-up

### PR DESCRIPTION
**Pull request by BachToTheFuture**
Some changes that come with this pull request:
* An updated Makefile
* Added a README
* Moved source files to the "src" directory
* Added a screenshot

### CODE REVIEW

**Comment by humdingerb** [comment]
To not let you off too easily for this task, how about cleaning up the source according to the [Haiku Coding Style](https://www.haiku-os.org/development/coding-guidelines)?  :)

**Comment by scottmc** [comment]
Also, once this is merged it'd be nice to also see a matching recipe at Haikuports for it.

**Comment by owenca** [comment]
> To not let you off too easily for this task, how about cleaning up the source according to the [Haiku Coding Style](https://www.haiku-os.org/development/coding-guidelines)? :)

@bach5000 You might want to use `haiku-format` found [here](https://github.com/owenca/haiku-format).

**Comment by BachToTheFuture** [comment]
@humdingerb @scottmc @owenca:
Cleaned up the source!

**Review by humdingerb — CHANGES_REQUESTED** [review]
* in RandomColor.cpp, l.29 ff: No need for the curly brackets, also remove empty lines

* Use Koder to find unneeded whitespaces.

* In TileSaver.cpp,
   l.104: too much indentation. Just one more than the previous line.
   l.106: too much indentation. Just one more than the previous line. Re-wrap to 80 chars.
   l.134: no need for the curly brackets

* The "*" of pointers belong to the type, e.g. `BView *pConfigView` -> `BView* pConfigView`

* Two blank lines between functions

**Comment by BachToTheFuture** [comment]
@humdingerb Ah. Will fix now :)

**Comment by BachToTheFuture** [comment]
@humdingerb I think I got all the things you mentioned in the review... Not sure though :)

File: src/TileSaver.cpp [hunk 1]
```diff
@@ -83,55 +88,56 @@ TileSaver::Draw(BView *pView, int32 frame)
...
+			B_FOLLOW_ALL, B_WILL_DRAW);
 	pCaption->SetText("TileSaver\n"
-							"\n"
-							"\t© 1999  Jens Kilian <jjk@acm.org>\n"
-							"\n"
-							"TileSaver comes with ABSOLUTELY NO WARRANTY; "
-							"it is free software, and you are welcome "
-							"to redistribute it under certain conditions.  "
-							"See the GNU General Public License for details.");
+				"\n"
```
Comment (humdingerb) [inline src/TileSaver.cpp]: You can indent all the text one tab more that line 110 and re-wrap to 80 chars.

File: src/TileSaver.cpp [hunk 2]
```diff
@@ -83,55 +88,56 @@ TileSaver::Draw(BView *pView, int32 frame)
...
+				"\t© 1999  Jens Kilian <jjk@acm.org>\n"
+				"\n"
+				"TileSaver comes with ABSOLUTELY NO WARRANTY;"
+				"it is free software, and you are welcome"
+				"to redistribute it under certain conditions."
```
Comment (humdingerb) [inline src/TileSaver.cpp]: Maybe add a "`\n`" at the end of this line.

**Review by humdingerb — CHANGES_REQUESTED** [review]
There are still a few stray tabs in TileSaver.cpp|h.
Also check indentation on TileSaver.h so the function/variable names line up. (4 spaces per tab)
I recommend Koder esp. for the whitespaces.

**Comment by BachToTheFuture** [comment]
@humdingerb May have to wait until night to be able to use Koder... but will fix :)

**Comment by BachToTheFuture** [comment]
@humdingerb You're right... I can see a lot of errors when I open these files up with Koder... yikes.
But I fixed the whitespace issues! So I'm going to push those commits right now.

**Comment by humdingerb** [comment]
Thanks. I did the re-wrapping of the config text myself. Hope you don't mind.

**Comment by BachToTheFuture** [comment]
@humdingerb Sorry for making you do work :(

**Pull request merged**